setup.py

```diff
@@ -186,7 +186,8 @@ def find_any_distribution(pkgs):
 
 
 build_chainerx = 0 != int(os.getenv('CHAINER_BUILD_CHAINERX', '0'))
-if os.getenv('READTHEDOCS', None) == 'True':
+if os.getenv('READTHEDOCS', None) == 'True' \
```

Comment:
Could you surround conditions `()` and avoid using `\`?

Reply:
I updated, thank you.